### DISCUSSION TITLE
does automa can click another extension chrome ?

### DISCUSSION THREAD

**Discussion by jobwriter91** [AutomaApp/automa#1257] [Q&A]
can i create workflow by attaching chrome extension to my workflow block?

**Comment by fmen1** [comment]
Nope, not that I know of anyway. The reason there is no straight forward way of doing so is for security reasons. If a workaround is possible, I too would like to know.

**Reply by whwh656** [reply]
> can i create workflow by attaching chrome extension to my workflow block?

litrpa can do it

**Reply by jobwriter91** [reply]
can muti language ? because cn language only? its free?

**Comment by jobwriter91** [comment]
i hope workflow can be fusion with other chrome extention

**Comment by fmen1** [comment]
I hope so too! Perhaps the developer of this incredible project, @Kholid060 , can give us his thoughts.

**Comment by fmen1** [comment]
Actually, there is some dicussion on this topic here:
https://github.com/AutomaApp/automa/discussions/500

**Comment by jobwriter91** [comment]
multi language?

Pada tanggal Jum, 12 Mei 2023 pukul 14.40 whwh656 ***@***.***>
menulis:
